.cg-docs/solutions/testing-patterns/2026-06-04-r-lambda-syntax-missed-by-function-regex.md

```
@@ -0,0 +1,74 @@
+
+## Solution
+
+Use an alternation pattern that covers both syntaxes when auditing:
+
+```regex
+(<- function)|(<- \\()
+```
+
+In `grep_search` (VS Code tool):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)